crypto/data_formats_test.go

```diff
@@ -16,14 +16,93 @@ package crypto
 
 import (
 	"bytes"
+	"reflect"
 	"testing"
 
 	"github.com/google/trillian"
 	"github.com/google/trillian/testonly"
+	"github.com/kr/pretty"
```

Comment:
This is not the `pretty` package we use everywhere else.

Reply:
Removed